doc/sphinxext/gallery_generator.py

```diff
@@ -240,9 +240,13 @@ def thumbfilename(self):
 
     @property
     def apiname(self):
-        name = self.modulename.split("_")
-        name = name[1::]
-        return "_".join(name)
+        name=""
+        with open(op.join(self.target_dir, self.pyfilename), "r") as file:
+            regex = r"az\.(plot\_[a-z_]+)\("
```

Comment:
This sounds ok. Do we have code that calls 

    from arviz import plot_xyz

Reply:
@ahartikainen Currently in all the example files, we are doing `import arviz as az` and then using the plot function `az.plot_xyz`.